account_chart_update_l10n_eu_oss/wizard/wizard_chart_update.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2022 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class WizardUpdateChartsAccounts(models.TransientModel):
+    _inherit = "wizard.update.charts.accounts"
+
+    def _domain_taxes_to_deactivate(self, found_taxes_ids):
+        domain = super()._domain_taxes_to_deactivate(found_taxes_ids)
+        domain.append(("tax_group_id.name", "not like", "OSS%%"))
```

Comment:
Better to use the specific field `oss_country_id`

Reply:
I cannot where that field is defined, could you point it to me?

Reply:
https://github.com/OCA/account-fiscal-rule/blob/14.0/l10n_eu_oss/models/account_tax.py#L10

But this is for the OCA module. Maybe you are not using it?

Reply:
The 15.0 reference: https://github.com/OCA/account-fiscal-rule/blob/15.0/l10n_eu_oss_oca/models/account_tax.py#L10

Reply:
It doesn't cover correctly B2C/B2B flows at the same time, as the auto-application of the fiscal positions can't cover correctly all the cases.

Reply:
I see, I think we need to cover both cases for those who have the Odoo one already installed. I can create another glue module `account_chart_update_l10n_eu_oss_oca` for the OCA alternative, WDYT?

Reply:
OCA, but Odoo link is very weak and error prone. I urge you to switch to OCA one.

Reply:
What I meant above is we can have both `account_chart_update_l10n_eu_oss_oca` and `account_chart_update_l10n_eu_oss` is this feasible for you?

> I urge you to switch to OCA one.

I'll try!

Reply:
Yes, of course.